index-lenient.d.ts

```diff
@@ -0,0 +1,10 @@
+import './index';
+
+declare global {
+  interface LenientGlobalVariableTypes {
+    canvas: never;
```

Comment:
I think instead of doing it for each variable individually, I would just one configuration value. We can still keep it for `canvas`, `game`, and `socket` for now, as we already have those, but moving forward, I think I'd prefer only a single configuration. We could do that breaking change when we move over to the V9 types.

Reply:
I don't agree. We already have one global switch with using this file. I'd like to keep this separately configurable for people that want to. There might be different reasons why someone is willing to take the risk for one variable, but not for another.

Reply:
My main argument here is that we probably want to expand this to properties of `game. For example, all of the collections in `Game` (like `Game#users` etc.) should probably be typed leniently if when the lenient configuration is used. Do you also want to configure all of them individually? Or Would that be done automatically if it's configured for `game`?

Reply:
That would be in line with how I did `ui`.